parsers/jscript.c

```diff
@@ -1741,10 +1775,12 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 					if (isKeyword (token, KEYWORD_async))
 						readToken (token);
 				}
+
+				vString * signature = NULL;
 				if ( is_shorthand || isKeyword (token, KEYWORD_function) )
 				{
 					TRACE_PRINT("Seems to be a function or shorthand");
-					vString *const signature = vStringNew ();
+					signature = vStringNew ();
```

Comment:
*1*, here. However, I found that this delayed memory allocation makes reading the code harder.
Even if we may not use the allocated object, how about allocating the object at ...*2*